server/app/services/applicant/ApplicantService.java

```diff
@@ -320,6 +325,68 @@ public CompletionStage<Application> submitApplication(
                     applicantId, programId, /* tiSubmitterEmail= */ Optional.empty()));
   }
 
+  /**
+   * Set the status for the newly submitted application to the given status.
+   *
+   * <p>Because this is done programmatically, we insert an event without an account attached to it.
+   * For now, this should only be used when setting the application status to the default status for
+   * the program.
+   *
+   * @param application the application on which to set the status
+   * @param status the status to set the application to
+   */
+  private void setApplicationStatus(Application application, StatusDefinitions.Status status) {
+    // Set the status for the application automatically to the default status
+    ApplicationEventRepository aer = new ApplicationEventRepository();
+    ApplicationEventDetails.StatusEvent statusEvent =
+        ApplicationEventDetails.StatusEvent.builder()
+            .setStatusText(status.statusText())
+            .setEmailSent(true)
+            .build();
+    ApplicationEventDetails details =
+        ApplicationEventDetails.builder()
+            .setEventType(ApplicationEventDetails.Type.STATUS_CHANGE)
+            .setStatusEvent(statusEvent)
+            .build();
+    // Because we are doing this automatically, set the Account to empty.
+    aer.insertSync(new ApplicationEvent(application, Optional.empty(), details));
+  }
+
+  /**
+   * Email the applicant with the status' defined email, if the applicant's email address exists and
+   * the status has an email associated with it.
+   *
+   * @param applicantId the ID of the applicant
+   * @param programDef the ProgramDefinition that the applicant applied for
+   * @param status the status from which to get the email body to send
+   */
+  private void maybeNotifyApplicantWithStatus(
```

Comment:
I don't love that I'm basically copying the code from ProgramAdminApplicationService here.  Maybe there's another place we can put this?

Reply:
Cool, I wasn't sure if it was too weird to call an admin service from the applicant service, but that'd be better than copying the code.

Reply:
This is probably sufficiently different now that it is async to leave it as is.